### DISCUSSION TITLE
Are the maintainers on holiday?

### DISCUSSION THREAD

**Discussion by robertvanhoesel** [ueberdosis/tiptap#1688] [General Questions] [answered]
I'm a bit surprised my PR ( https://github.com/ueberdosis/tiptap/pull/1540 ) for a bug fix from over a month ago hasn't received any comments or feedback from the maintainers. Likewise, I would appreciate this PR ( https://github.com/ueberdosis/tiptap/pull/1556 ) to be merged as it is a legit improvement to TipTap. 

After a few attempts to get your attention, even reaching out personally, there is complete radio silence. 

I can absolutely understand if you are 1) too busy/on leave, 2) of the opinion the PR doesn't fit your model/should be reworked or 3) it conflicts with other roadmap items or prios, but the least you can do is respond/comment to PRs and issues.

<details>
<summary>Rant</summary>
I mean what the f should it take to get a bug fixed? I created an Issue (#1518) with a very detailed repro sandbox,  other people pitch in they are facing the same issue – also on discord, I create a PR with additional tests etc. Heck, the issue even has a nice 'Sponsor' badge next to it... guess that is worthless.
</details>

I would really appreciate a review, reply or any form of communication.

**Comment by hanspagel** [comment, marked as answer]
Holiday and life came in between, sorry for the long wait. We do our best to get back to everyone personally. Expect us to be get more responsive starting next week.

**Reply by robertvanhoesel** [reply]
Thanks for the response – we all need a break from time to time 👍 . Looking forward to it!
